frontend-react/src/components/shared/Table/Table.tsx

```diff
@@ -0,0 +1,225 @@
+    const [sortOrder, setSortOrder] = useState(FilterOptionsEnum.NONE);
+    const sortedData = useMemo(() => {
+        return sortOrder !== FilterOptionsEnum.NONE && activeColumn
+            ? rowData.sort((a, b): number => {
+                  let contentColA = convertStringToSortableRepresentation(
+                      a.find((item) => item.columnKey === activeColumn)?.content
+                  );
+                  let contentColB = convertStringToSortableRepresentation(
+                      b.find((item) => item.columnKey === activeColumn)?.content
+                  );
+
+                  if (sortOrder === FilterOptionsEnum.ASC) {
+                      return contentColA < contentColB ? 1 : -1;
+                  } else {
+                      return contentColA > contentColB ? 1 : -1;
+                  }
+              })
+            : rowData;
+    }, [activeColumn, rowData, sortOrder]);
+    return (
+        <>
+            <thead>
+                <tr>
+                    {columnHeaders.map((header) => {
+                        return (
+                            <SortableTableHeader
+                                columnTitle={header}
+                                activeColumn={activeColumn}
+                                sortOrder={sortOrder}
+                                setActiveColumn={setActiveColumn}
+                                setSortOrder={setSortOrder}
+                            />
+                        );
+                    })}
+                </tr>
+            </thead>
+            <tbody>
+                {sortedData.map((row) => {
+                    return (
+                        <tr>
+                            {row.map((data) => {
+                                return (
+                                    <td
+                                        dangerouslySetInnerHTML={{
+                                            __html: sanitizeHtml(data.content),
+                                        }}
+                                    />
+                                );
+                            })}
+                        </tr>
+                    );
+                })}
+            </tbody>
+        </>
+    );
+};
+
+export const Table = ({
+    borderless,
+    compact,
+    emptyState,
+    fullWidth,
+    scrollable,
+    sortable,
+    stackedStyle,
+    striped,
+    rowData,
+}: TableProps) => {
+    const classes = classnames("usa-table", {
+        "usa-table--borderless": borderless,
+        "usa-table--compact": compact,
+        "width-full": fullWidth,
+        "usa-table--stacked": stackedStyle === "default",
+        "usa-table--stacked-header": stackedStyle === "headers",
+        "usa-table--striped": striped,
+    });
+
+    const columnHeaders = [
+        ...new Set(rowData.flat().map((data: RowData) => data.columnKey)),
+    ];
+
+    return (
+        <div
+            className={styles.reportstreamTable}
```

Comment:
Let's use `classnames` for variable class names.

Reply:
Can you tell me more? This is a CSS Module reference.